plenum/test/checkpoints/test_backup_replica_resumes_ordering_on_lag_in_checkpoints.py

```diff
@@ -98,11 +97,9 @@ def test_backup_replica_resumes_ordering_on_lag_in_checkpoints(
 
     # Ensure that the replica has adjusted last_ordered_3pc to the end
     # of the last checkpoint
-    def chk(r):
-        print(r)
-        assert r.last_ordered_3pc == (view_no, (Replica.STASHED_CHECKPOINTS_BEFORE_CATCHUP + 1) * CHK_FREQ)
     looper.run(
-        eventually(chk,
+        eventually(lambda *args: assertExp(slow_replica.last_ordered_3pc == \
+                        (view_no, (Replica.STASHED_CHECKPOINTS_BEFORE_CATCHUP + 1) * CHK_FREQ)),
```

Comment:
- `args` doesn't seem to be used in lambda, so why use arguments at all?
- there is `assert_eq` helper which shows left-hand and right-hand values when assert fails